src/get-modules-codemod-path.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const modulesCodemodRelPath = 'ember-modules-codemod/bin/ember-modules-codemod.js';
+const modulesCodemodChildPath = path.join(__dirname, '../node_modules', modulesCodemodRelPath);
+const modulesCodemodSiblingPath = path.join(__dirname, '../../', modulesCodemodRelPath);
+
+module.exports = function getModulesCodemodPath() {
```

Comment:
Does `require.resolve('ember-modules-codemod')` give anything meaningful that we can use?

Reply:
No, its `package.json` has no `main` entry, so we can't `require` it :( We could do some hackery where get the module search path and traverse it manually I guess...